docs/backend/native_api.ipynb

```diff
@@ -0,0 +1,324 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Native Server API\n",
+    "\n",
+    "Apart from the OpenAI compatible API, the SGLang Runtime also provides its native server API. We introduce these following API:\n",
+    "\n",
+    "- `/generate`\n",
+    "- `/update_weights`\n",
+    "- `/get_server_args`\n",
+    "- `/get_model_info`\n",
+    "- `/health`\n",
+    "- `/health_generate`\n",
+    "- `/flush_cache`\n",
+    "- `/get_memory_pool_size`\n",
+    "\n",
+    "We mainly use `requests` to test these APIs in the following examples. You can also use `curl`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Launch The Server"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sglang.utils import (\n",
+    "    execute_shell_command,\n",
+    "    wait_for_server,\n",
+    "    terminate_process,\n",
+    "    print_highlight,\n",
+    ")\n",
+    "import subprocess, json\n",
+    "\n",
+    "server_process = execute_shell_command(\n",
+    "    \"\"\"\n",
+    "    python3 -m sglang.launch_server --model-path meta-llama/Llama-3.2-1B-Instruct --port=30010\n",
+    "\"\"\"\n",
+    ")\n",
+    "\n",
+    "wait_for_server(\"http://localhost:30010\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Generate\n",
+    "\n",
+    "Used to generate completion from the model, similar to the `/v1/chat/completions` API in OpenAI."
```

Comment:
similar to `v1/completion`, not the chat one.
Point them to https://sgl-project.github.io/references/sampling_params.html for argument specifications.